packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -70,53 +74,18 @@ export class CallContext {
     return this._state;
   }
 
-  public setState(state: CallClientState): void {
-    this._state = state;
+  public modifyState(modifier: (draft: CallClientState) => void): void {
```

Comment:
I don't understand how this function handling batching now that we are getting rid of the batch method?

Reply:
Nothing was using `batch()` before this PR either. The only caller of `batch` was `modifyState`, and nobody was calling `modifyState`.

`modifyState` was originally intended to be exposed as part of the public API, but we decided against it pre-GA, so I'm cleaning up that dead code. The `modifyState` I have _introduced_ in this PR is actually just `setState`, but with the call to `produce` folded in.

So, it actually reduces the boilerplate, without *any logical changes whatsoever*.